exporter/loggingexporter/factory.go

```diff
@@ -43,28 +44,54 @@ func (f *Factory) CreateDefaultConfig() configmodels.Exporter {
 			TypeVal: typeStr,
 			NameVal: typeStr,
 		},
+		LogLevel: "info",
 	}
 }
 
-func noopStopFunc() error {
-	return nil
-}
-
 // CreateTraceExporter creates a trace exporter based on this config.
-func (f *Factory) CreateTraceExporter(logger *zap.Logger, cfg configmodels.Exporter) (consumer.TraceConsumer, exporter.StopFunc, error) {
+func (f *Factory) CreateTraceExporter(logger *zap.Logger, config configmodels.Exporter) (consumer.TraceConsumer, exporter.StopFunc, error) {
+	cfg := config.(*Config)
 
-	lexp, err := NewTraceExporter(cfg.Name(), logger)
+	le, err := f.createLoggingExporter(logger, cfg.LogLevel)
 	if err != nil {
 		return nil, nil, err
 	}
-	return lexp, noopStopFunc, nil
+
+	lexp, err := NewTraceExporter(cfg.Name(), le.logger)
+	if err != nil {
+		return nil, nil, err
+	}
+	return lexp, le.stop, nil
+}
+
+func (f *Factory) createLoggingExporter(logger *zap.Logger, logLevel string) (*loggingExporter, error) {
```

Comment:
Is the parameter `logger *zap.Logger` used? I think it can be removed. Also the return value can be `*zap.Logger` instead of `*loggingExporter`.

```suggestion
func (f *Factory) createLoggingExporter(logLevel string) (*zap.Logger, error) {
```